### PR TITLE
Added Docs Project Area

### PR DESCRIPTION
**Pull request by awanlin**
## Hey, I just made a Pull Request!

Added the Documentation Project Area with @aramissennyeydd, @waldirmontoya25, and myself as initial members

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: OWNERS.md [hunk 1]
```diff
@@ -158,6 +158,18 @@ Scope: The Scaffolder frontend and backend plugins, and related tooling.
...
+### Documentation
+
+Team: @backstage/documentation-maintainers
+
+Scope: The Backstage Documentation
+
+| Name           | Organization | GitHub                                                | Discord           |
+| -------------- | ------------ | ----------------------------------------------------- | ----------------- |
+| Andre Wanlin   | Spotify      | [awanlin](https://github.com/awanlin)                 | `ahhhndre`        |
+| Aramis Sennyey |              | [aramissennyeydd](https://github.com/aramissennyeydd) | `Aramis#7984`     |
```
Comment (aramissennyeydd) [inline OWNERS.md:170]: Note for myself, I'll adjust my org in a followup PR

**Review by aramissennyeydd — APPROVED** [review]
Thanks for setting this up!

File: OWNERS.md [hunk 2]
```diff
@@ -158,6 +158,18 @@ Scope: The Scaffolder frontend and backend plugins, and related tooling.
...
+| Waldir Montoya | Broadcom     | [waldirmontoya25](https://github.com/waldirmontoya25) | `waldirmontoya25` |
```
Comment (Rugvip) [inline OWNERS.md:171]: Perhaps we could skip Waldir for now if you want to ship this sooner?

**Comment by joshjung** [comment]
I would like to be added to this, per our discussions @Rugvip and @ahhhndre. You can put my company as Vail Systems.

**Comment by Parsifal-M** [comment]
Happy to be added (@aramissennyeydd, thanks!) you can add me as part of VodafoneZiggo 👍

**Comment by awanlin** [comment]
Following up that I'll be working on updating this on this Friday 👍

**Comment by awanlin** [comment]
Hi @joshjung, I'm happy to add you in a follow up PR. Currently you need to be an active [Org Member](https://github.com/backstage/community/blob/main/GOVERNANCE.md#organization-member) to be part of a Project Area. Given how early stages we are with this not being part of the project area won't really impact you. There's lots of decisions and thinkings that need to be done that you can actively take part in. 👍

**Comment by awanlin** [comment]
Hi @Parsifal-M, I was finally getting back to this and it seems you aren't an [Org Member](https://github.com/backstage/community/blob/main/GOVERNANCE.md#organization-member) yet either, you need to be one to be part of a Project Area. You can still take part in SIGs though. You might already meet the requirements, can you apply, please? Once that's done I can add you.

**Comment by Parsifal-M** [comment]
> Hi @Parsifal-M, I was finally getting back to this and it seems you aren't an [Org Member](https://github.com/backstage/community/blob/main/GOVERNANCE.md#organization-member) yet either, you need to be one to be part of a Project Area. You can still take part in SIGs though. You might already meet the requirements, can you apply, please? Once that's done I can add you.

Ah clear! Will do that on Monday, no worries!

**Comment by awanlin** [comment]
Updated this PR, I'm assuming that this will be pending approval of this [Org Member request](https://github.com/backstage/community/issues/139).

**Comment by awanlin** [comment]
@Parsifal-M now that you are an Org Member can you approve this, please?

**Review by Parsifal-M — APPROVED** [review]
Looks good! 👏

**Review by freben — APPROVED** [review]
🎉🎉🎉

**Pull request merged**